docker-plugin/src/test/java/com/nirima/jenkins/plugins/docker/DockerTemplate2Test.java

```diff
@@ -21,6 +19,33 @@
 
     @Test
     @LocalData
+    @Issue("jenkinsci/docker-plugin/issues/263")
+    public void load080Config() {
+        final DockerCloud dock = (DockerCloud) j.getInstance().getCloud("someCloud");
+        assertThat(dock.getTemplates(), hasSize(6));
+
+        final DockerTemplate template = dock.getTemplate("somedomain.com:5000/dockerbuild");
+        assertThat(template, notNullValue());
+
+        assertThat(template.getLabelString(), equalTo("dockerbuild-local"));
+        assertThat(template.remoteFs, equalTo("/home/jenkins"));
+
+        final DockerTemplateBase tBase = template.getDockerTemplateBase();
+        assertThat(tBase, notNullValue());
+
+        assertThat(tBase.getDockerCommandArray()[0], equalTo("/usr/bin/start-jenkins-slave.sh"));
+        assertThat(tBase.getDnsString(), equalTo("8.8.8.8"));
+
+        assertThat(tBase.getVolumesString(), equalTo("/dev/log:/dev/log"));
+        assertThat(asList(tBase.getVolumes()), hasSize(1));
+        assertThat(tBase.getVolumes()[0], equalTo("/dev/log:/dev/log"));
+
+        assertFalse(tBase.bindAllPorts);
```

Comment:
they are all from static config and method is pretty simple - validate loaded data. I usually set debugger breakpoint and navigate objects/errors.